requirements_searching.py

```diff
@@ -0,0 +1,44 @@
+import os
+import re
+
+
+def _find_files(path, pattern):
+    matching_files = []
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            if pattern.match(file):
+                relative_path = os.path.relpath(os.path.join(root, file), start=path)
+                matching_files.append(relative_path)
+    return matching_files
+
+
+def _get_file_names(files):
+    file_names = [os.path.split(file)[-1] for file in files]
+    return file_names
+
+
+def _convert_paths_to_linux(paths):
+    return [path.replace("\\", "/") for path in paths]
+
+
+def use_requirements(path):
+    pattern = re.compile(r".*requirements.*", re.IGNORECASE)
```

Comment:
Case in filenames matters on Linux, it should be avoided

Reply:
Sure, I will delete this.